php_phongo.c

```diff
@@ -451,10 +451,63 @@ mongoc_bulk_operation_t *phongo_bulkwrite_init(zend_bool ordered) { /* {{{ */
 #define PHONGO_WRITECONCERN_ALLOWED   0x01
 #define PHONGO_READPREFERENCE_ALLOWED 0x02
 
-static int process_read_preference(zval *option, zval **zreadPreference TSRMLS_DC)
+static int process_read_concern(zval *option, bson_t *mongoc_opts TSRMLS_DC)
+{
+	if (Z_TYPE_P(option) == IS_OBJECT && instanceof_function(Z_OBJCE_P(option), php_phongo_readconcern_ce TSRMLS_CC)) {
+		const mongoc_read_concern_t *read_concern = phongo_read_concern_from_zval(option TSRMLS_CC);
+
+		mongoc_read_concern_append((mongoc_read_concern_t*)read_concern, mongoc_opts);
```

Comment:
`mongoc_read_concern_append()` returns a success boolean. If false, you should throw an exception saying we couldn't append it properly.

Reply:
done